Floofbot/Modules/Administration.cs

```diff
@@ -421,105 +422,50 @@ await foreach(var messageCollection in asyncMessageCollections)
         [Alias("wl")]
         [Summary("Displays the warning log for a given user")]
         [RequireContext(ContextType.Guild)]
```

Comment:
Kind of a bummer this prevents usage of `warnlog` in a DM with the bot. But adding logic to check if the first argument is a guild ID/name is going to be a bit messy and is more work.